modules/apps/object/object-service/src/main/java/com/liferay/object/internal/bulk/selection/DeleteObjectBulkSelectionAction.java

```diff
@@ -67,22 +78,46 @@ public void execute(
 						if (object instanceof ObjectEntry) {
 							ObjectEntry objectObjectEntry = (ObjectEntry)object;
 
-							_objectEntryLocalService.deleteObjectEntry(
-								objectObjectEntry);
+							ObjectDefinition objectDefinition =
+								_objectDefinitionLocalService.
+									getObjectDefinition(
+										objectObjectEntry.
+											getObjectDefinitionId());
+
+							ObjectEntryManager objectEntryManager =
```

Comment:
If possible, can you use the pattern below? It already encapsulates the UnsupportedOperationException rule.

```
DefaultObjectEntryManager defaultObjectEntryManager =
	DefaultObjectEntryManagerProvider.provide(
		_objectEntryManagerRegistry.getObjectEntryManager(
			_objectDefinition.getCompanyId(),
			_objectDefinition.getStorageType()));
```